code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -388,7 +388,7 @@
 		return REACTING
 
 /datum/gas_reaction/nobliumformation //Hyper-Noblium formation is extrememly endothermic, but requires high temperatures to start. Due to its high mass, hyper-nobelium uses large amounts of nitrogen and tritium. BZ can be used as a catalyst to make it less endothermic.
-	priority = 6
+	priority = 5001 // Skyrat edit: priority changed so noblium doesn't stop noblium formation
```

Comment:
this is intentional and prevents hyper-noblium from making--literally--1.5 million science points whenever anyone bothers to make it

Reply:
Ah, alright. Removed it.